### PR TITLE
EPMDJ-4840: Update build/scope test table

### PR DESCRIPTION
**Pull request by IliaSolovev**
Issue: EPMDJ-4840

### CODE REVIEW

File: src/pages/agent-full-page/coverage-plugin/test-details/test-details.tsx [hunk 1]
```diff
@@ -112,5 +116,5 @@ export const TestDetails = testDetails(
 );
 
 const Title = testDetails.title(Panel);
-const TableCell = testDetails.tableCell('span');
+const TestTypeCell = testDetails.testTypeCell('span');
```
Comment (RomanAchilov) [inline src/pages/agent-full-page/coverage-plugin/test-details/test-details.tsx:119]: we also want to unify the Cells used

File: src/pages/agent-full-page/coverage-plugin/test-details/test-details.tsx [hunk 2]
```diff
@@ -112,5 +116,5 @@ export const TestDetails = testDetails(
 );
 
 const Title = testDetails.title(Panel);
-const TableCell = testDetails.tableCell('span');
+const TestTypeCell = testDetails.testTypeCell('span');
 const StatusCell = testDetails.statusCell('span');
```
Comment (RomanAchilov) [inline src/pages/agent-full-page/coverage-plugin/test-details/test-details.tsx:120]: we also want to unify the Cells used

File: src/pages/agent-full-page/coverage-plugin/test-details/test-details.tsx [hunk 3]
```diff
@@ -35,30 +35,31 @@ export const TestDetails = testDetails(
...
+              idKey="testName"
               columnsSize="medium"
-              expandedContentKey="tests"
             >
               <Column
                 name="testName"
                 label="Name"
                 Cell={({ item: { id, testName } }) => (
-                  <Cells.Compound cellName={testName} cellAdditionalInfo={id} icon={<Icons.Test height={16} width={16} />} />
+                  <Cells.Compound cellName={testName} cellAdditionalInfo={id || '-'} icon={<Icons.Test height={16} width={16} />} />
```
[the remark this answers is not in the document]
Comment (RomanAchilov) [reply src/pages/agent-full-page/coverage-plugin/test-details/test-details.tsx:45]: `-` replace `&ndash;`

**Review by RomanAchilov — CHANGES_REQUESTED** [review]

File: src/components/cells/status-cell/status-cell.module.scss [hunk 4]
```diff
@@ -0,0 +1,19 @@
+@import '~common/style-guide/index';
+
+.status-cell {
+  font-family: $semibold-font;
+  font-size: 12px;
```
Comment (RomanAchilov) [inline src/components/cells/status-cell/status-cell.module.scss:5]: default

File: src/components/cells/status-cell/status-cell.module.scss [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+  line-height: 16px;
```
Comment (RomanAchilov) [inline src/components/cells/status-cell/status-cell.module.scss:6]: default

File: src/components/cells/status-cell/status-cell.tsx [hunk 6]
```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { BEM } from '@redneckz/react-bem-helper';
+
+import { capitalize } from '../../../utils';
```
Comment (RomanAchilov) [inline src/components/cells/status-cell/status-cell.tsx:4]: from 'utils'

File: src/components/cells/status-cell/status-cell.tsx [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+import styles from './status-cell.module.scss';
+
+interface Props {
+  className?: string;
+  value?: string;
+}
+
+const statusCell = BEM(styles);
+
+export const StatusCell = statusCell(({ className, value }: Props) => (
```
Comment (RomanAchilov) [inline src/components/cells/status-cell/status-cell.tsx:15]: export const StatusCell = BEM(styles)("div")

**Review by RomanAchilov — CHANGES_REQUESTED** [review]

File: src/components/cells/status-cell/status-cell.module.scss [hunk 8]
```diff
@@ -0,0 +1,13 @@
+@import '~common/style-guide/index';
+
+.status-cell {
+  font-family: $semibold-font;
+  &--type {
```
Comment (RomanAchilov) [inline src/components/cells/status-cell/status-cell.module.scss:5]: empty line

**Review by RomanAchilov — APPROVED** [review]

**Review by pavyarov — APPROVED** [review]

**Pull request merged**